types/single-spa-react.d.ts

```diff
@@ -1,21 +1,21 @@
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import * as ReactDOMClient from "react-dom/client";
 import { AppProps, CustomProps, LifeCycleFn } from "single-spa";
 
 export const SingleSpaContext: React.Context<CustomProps & AppProps>;
 
+export interface ReactAppOrParcel<ExtraProps> {
```

Comment:
Why is this needed? Could we use single-spa's `Lifecycles` type instead?

https://github.com/single-spa/single-spa/blob/80682e10ec929644d7a50280ea0d43675d31ee56/typings/single-spa.d.ts#L56